playbooks/galaxy.yaml

```
@@ -24,7 +24,7 @@
           CONN_MAX_AGE: 0
           OPTIONS:
             sslmode: '{{ postgres_sslmode }}'
-            sslrootcert: "{{'/etc/pki/tls/certs/ca-bundle.crt' if (postgres_sslmode in ['verify-ca', 'verify-full'])}}"
+            sslrootcert: "{{'/etc/pki/tls/certs/ca-bundle.crt' if (postgres_sslmode in ['verify-ca', 'verify-full']) else ''}}"
```

`sslrootcert` will be set to an empty string when not using verify-{ca,full}.
Instead you can use the ternary filter to avoid adding `sslrootcert` to the database settings
```suggestion
            sslrootcert: "{{ (postgres_sslmode in ['verify-ca', 'verify-full']) | ternary('/etc/pki/tls/certs/ca-bundle.crt', omit) }}"
```

---

An empty string will behave the same as omitting the sslrootcert parameter. According to the [docs](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNECT-SSLROOTCERT), the server's certificate will only be verified if the file exists.